src/Tests/Microsoft.NET.TestFramework/TestAssetsManager.cs

```diff
@@ -52,6 +52,15 @@ public TestAssetsManager(ITestOutputHelper log)
             return testAsset;
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="testProject">The testProject used to create a teatAsset with.</param>
+        /// <param name="callingMethod">Defaults to the name of the caller function (presumably the test). Used to prevent file collisions on tests which share the same test project.</param>
+        /// <param name="identifier">An ID for the project that should be unique for every test, including each theory child test.
```

Comment:
The ID doesn't need to be different for tests where the `callingMethod` is different.

```suggestion
        /// <param name="identifier">An ID that should be different for every child test of a theory, to ensure that each test uses a different test folder.
```